### PR TITLE
Added support for Python3

### PR DESCRIPTION
**Pull request by philippmeisberger**
Removed import of deprecated package Image: Instead importing from package PIL; Ready to support Python3 and still supporting Python2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Than clone this repository:
 Build the package:
 
     ~$ cd ./pyfingerprint/src/
-    ~$ debuild
+    ~$ dpkg-buildpackage -uc -us
```
Comment (bastianraschke) [inline README.md:23]: :+1:

File: src/debian/authors [hunk 2]
```diff
@@ -1,2 +1,3 @@
 Bastian Raschke <bastian.raschke@posteo.de>
+Philipp Meisberger <team@pm-codeworks.de>
```
Comment (bastianraschke) [inline src/debian/authors:2]: Please add yourself into the "debian/copyright" file as well.

File: src/debian/changelog [hunk 3]
```diff
@@ -1,3 +1,15 @@
+python-fingerprint (1.5) UNRELEASED; urgency=medium
+
+  * Added support for Python3
+    * Still compatible with Python2
```
Comment (bastianraschke) [inline src/debian/changelog]: Better:
* Added support for Python 3 (still compatible with Python 2)

File: src/debian/changelog [hunk 4]
```diff
@@ -1,3 +1,15 @@
...
+  * storeTemplate() can be used without arguments
+    * The template is stored at the next free index automatically
+    * BREAKING CHANGE: The index is returned! In previous releases this was a boolean value.
+  * Introduced getStorageCapacity() to retrieve the sensor template capacity
+  * Introduced generateRandom() which generates a random number on the builtin hardware PRNG of the ZFM
+
+ -- Philipp Meisberger <team@pm-codeworks.de>  Sun, 09 Apr 2017 21:46:27 +0200
```
Comment (bastianraschke) [inline src/debian/changelog]: Wrong timezone i guess ;)

File: src/debian/control [hunk 5]
```diff
@@ -4,14 +4,31 @@ Section: python
...
+Depends: ${python:Depends},
+         ${misc:Depends},
          python-serial,
-         python-imaging,
-         ${python:Depends}, ${misc:Depends}
+         python-pil
+Architecture: all
+Description: Python written library for using the ZFM-20 fingerprint sensor
+ This library is required for using the ZhianTec ZFM-20 fingerprint sensor
+ (known as "Arduino fingerprint sensor") in Python.
```
Comment (bastianraschke) [inline src/debian/control]: Python 2

File: src/debian/control [hunk 6]
```diff
@@ -4,14 +4,31 @@ Section: python
...
+
+Package: python3-fingerprint
+Depends: ${python3:Depends},
+         ${misc:Depends},
+         python3-serial,
+         python3-pil
 Architecture: all
 Description: Python written library for using the ZFM-20 fingerprint sensor
  This library is required for using the ZhianTec ZFM-20 fingerprint sensor
```
Comment (bastianraschke) [inline src/debian/control]: And here Python 3 for description

File: src/debian/rules [hunk 7]
```diff
@@ -1,4 +1,20 @@
 #!/usr/bin/make -f
 
 %:
-	dh $@ --with python2
+	dh $@ --with python2,python3
```
Comment (bastianraschke) [inline src/debian/rules:4]: :+1:

File: README.md [hunk 8]
```diff
@@ -20,7 +20,7 @@ Than clone this repository:
...
 
 ## Installation
```
Comment (bastianraschke) [inline README.md:25]: Maybe add 2 installation sections for Python 2 and Python 3?

**Comment by bastianraschke** [comment]
I would be happy, if you provide a example script for the implemented generate_random functionality ;)

File: src/debian/changelog (hunk 4 above)
Comment (philippmeisberger) [reply src/debian/changelog]: My UTC is currently 16:48. Standard time is 18:48, so UTC +2 hours is correctly ;) I did not insert this time manually: "dch" program did it^^

**Pull request merged**